raster/src/test/scala/geotrellis/raster/io/geotiff/reader/GeoTiffReaderSpec.scala

```diff
@@ -349,11 +351,17 @@ class GeoTiffReaderSpec extends FunSpec
     val MeanEpsilon = 1e-8
 
     def testMinMaxAndMean(min: Double, max: Double, mean: Double, file: String) {
-      val SinglebandGeoTiff(tile, extent, _, _, _, _) = SinglebandGeoTiff(file)
+      import geotrellis.raster.summary.polygonal.Implicits._
```

Comment:
are not these implicits imported somehow netter (like via `geotrellis.raster.summary.polygonal._` or `geotrellis.raster.summary._`)?

Reply:
I suppose I might have been thinking that importing here avoided polluting the namespace everywhere when the extra methods are only required here. But I suppose that's not really a concern. In any case I'm pretty sure the imports could use some cleanup -- it's on my list of things to tackle before finally merging this.

Reply:
Updated with this line deleted. I added the Implicits to the package.scala awhile back so this import is superfluous.